lib/utils/index.js

```diff
@@ -554,6 +554,7 @@ const URL_KEYS = [
   'cloud_name',
   'cname',
   'format',
+  'long_url_signature',
```

Comment:
Since this URL represents a Boolean, 
can we rename it to something that reflects that?

Something like use_long_url_signature maybe?

Reply:
The variable name was defined in the task description. We did not change it to comply with the spec and with all the other SDKs which use the same variable name.

From the spec:

<img width="797" alt="CleanShot 2020-04-14 at 15 20 13@2x" src="https://user-images.githubusercontent.com/1785648/79224286-81a29780-7e63-11ea-8fe5-e2533f7c4d32.png">